site/_includes/partials/pagination.njk

```diff
@@ -23,23 +23,23 @@ element for the pagination (stock) navigation #}
   {% if paginationObject.href.previous %}
     <a class="pagination__button type--small gap-right-200 text-align-center" href="{{ paginationObject.href.previous }}">
       {{ icon('arrow-back', {hidden: true}) }}
-      <span>{{ 'i18n.common.newer_posts' | i18n(locale) }}</span>
```

Comment:
This seems semi-unrelated, - unless you're thinking that "posts" is a bad name for a paginated author page?

Reply:
Basically yea, so instead of newer or older, its next and back.